crates/networking/rpc/engine/fork_choice.rs

```diff
@@ -50,10 +45,20 @@ impl RpcHandler for ForkChoiceUpdatedV3 {
         if params.len() != 2 {
             return Err(RpcErr::BadParams("Expected 2 params".to_owned()));
         }
+
+        let forkchoice_state: ForkChoiceState = serde_json::from_value(params[0].clone())?;
+        // if there is an error when parsing, set to None
```

Comment:
To avoid having the Result, if there is a parsing error I set the payload attribute to None. This is not spec compliant, but we should find another way of dealing with this case without changing the type.